app/client/survey_admin/survey_admin.jade

```diff
@@ -1,18 +1,31 @@
 template(name="survey_admin")
-  .page-header
-    nav.navbar
-      .navbar-header
-        a.navbar-brand.pull-left(href="/")= survey.title
-      ul.nav.navbar-nav.survey-admin-nav
-        li
-          a(href="/admin/surveys/#{survey._id}/details" class=detailsActive) Details
-        li
-          a(href="/admin/surveys/#{survey._id}/forms" class=formsActive) Forms
-        li
-          a(href="/admin/surveys/#{survey._id}/users" class=usersActive) Users
-  if survey
-    if detailsActive
-      p= survey.description
-    else
-      if formsActive
-        +survey_admin_forms survey=survey
+  .survey-admin
+    nav.navbar.navbar-default.survey-admin--nav
+      .container-fluid
+        .navbar-header
+          h2= survey.title
+        ul.nav.navbar-nav.navbar-right
+          li(class="{{isActiveRoute 'survey_admin_details'}}")
+            a.survey-admin--view-link(href="{{pathFor 'survey_admin_details' id=id}}" data-page='details') Details
+          li(class="{{isActiveRoute regex='survey_admin_forms|survey_admin_forms_new'}}")
+            a.survey-admin--view-link(href="{{pathFor 'survey_admin_forms' id=id}}" data-page='forms') Forms
+          li(class="{{isActiveRoute 'survey_admin_users'}}")
+            a.survey-admin--view-link(href="{{pathFor 'survey_admin_users' id=id}}" data-page='users') Users
+
+    .container-fluid
+      if Template.subscriptionsReady
+
+        if isActiveRoute 'survey_admin_details'
+          +survey_admin_details survey=survey
+
+        else if isActiveRoute 'survey_admin_forms'
+          +survey_admin_forms survey=survey
+
+        else if isActiveRoute 'survey_admin_forms_new'
+          +survey_admin_forms_new survey=survey
+
+        else if isActiveRoute 'survey_admin_form_details'
+          +survey_admin_form_details
+
+        else if isActiveRoute 'survey_admin_users'
+          +survey_admin_users survey=survey
```

Comment:
@arnoldbm I tried this approach. It may work but I struggled when trying to keep track of the state of the admin area (current page). I suppose we will need to keep this in a reactive or session variable (using a session variable may help—I was trying to use a reactiveVar). And, since the form details/questions are so nested, I had trouble handing down the formId. 
I suppose we can refactor later.